core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -307,7 +307,8 @@ class KafkaRaftManager[T](
       time,
       discoverBrokerVersions,
       apiVersions,
-      logContext
+      logContext,
+      MetadataRecoveryStrategy.NONE
```

Comment:
Rather than change all these callers, couldn't we retain the original constructor for these?

Reply:
I originally did this, but after [this discussion](https://github.com/apache/kafka/pull/13277#discussion_r1607019116) reverted to the current approach